### PR TITLE
Modules: add `See also` section and other minor content changes

### PR DESCRIPTION
**Pull request by ppcano**
I wanted to include the `See also` section to mention the TypeScript template project.

I ended up doing some other minor changes. 

I've reviewed and I didn't want to change more, but I think the structure of this guide could be updated.

### CODE REVIEW

File: src/data/markdown/docs/01 guides/02 Using k6/07 Modules.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ excerpt: ''
 While writing test scripts, it's common to import different modules, or part of modules, for
 usage throughout the script. In k6, it is possible to import three different kinds of modules:
 
+- [Built-in modules](#built-in-modules)
+- [Local filesystem modules](#local-filesystem-modules)
+- [Remote HTTP(S) modules](#remote-http-s-modules)
+
 ### Built-in modules
```
Comment (simskij) [inline src/data/markdown/docs/01 guides/02 Using k6/07 Modules.md:15]:
```suggestion
## Built-in modules
```

File: src/data/markdown/docs/01 guides/02 Using k6/07 Modules.md [hunk 2]
```diff
@@ -26,7 +30,19 @@ with k6, the module itself may only use relative or absolute filesystem imports
...
+```js
+//my-test.js
+import { someHelper } from './helpers.js';
+
+export default function () {
+  someHelper();
+}
 ```
 
 ### Remote HTTP(S) modules
```
Comment (simskij) [inline src/data/markdown/docs/01 guides/02 Using k6/07 Modules.md:48]:
```suggestion
## Remote HTTP(S) modules
```

File: src/data/markdown/docs/01 guides/02 Using k6/07 Modules.md [hunk 3]
```diff
@@ -38,14 +54,17 @@ it in a test script**.
 
 ```js
 import { randomItem } from 'https://jslib.k6.io/k6-utils/1.0.0/index.js';
+
+export default function () {
+  randomItem();
+}
 ```
 
 ## Bundling node modules
```
Comment (simskij) [inline src/data/markdown/docs/01 guides/02 Using k6/07 Modules.md:63]:
```suggestion
# Bundling node modules
```

File: src/data/markdown/docs/01 guides/02 Using k6/07 Modules.md [hunk 4]
```diff
@@ -384,3 +354,19 @@ Note that on Windows, you also need to make sure that your drive in question, sa
 has been marked for sharing in the Docker settings:
 
 ![Running k6 in docker on Windows](images/Modules/running-k6-in-docker-on-windows.png)
+
+## The JSLib repository
```
Comment (simskij) [inline src/data/markdown/docs/01 guides/02 Using k6/07 Modules.md:358]:
```suggestion
### The JSLib repository
```

I would also move this under "Remote HTTP(S) modules"

File: src/data/markdown/docs/01 guides/02 Using k6/07 Modules.md [hunk 5]
```diff
@@ -384,3 +354,19 @@ Note that on Windows, you also need to make sure that your drive in question, sa
...
+
+[JSLib](https://jslib.k6.io/) is a set of libraries known to play nicely with k6, and available at https://jslib.k6.io/.
+
+- `k6-utils`
+- `jsonpath`
+- `form-urlencoded`
+- `papaparse`
```
Comment (simskij) [inline src/data/markdown/docs/01 guides/02 Using k6/07 Modules.md:365]:
```suggestion
```

I would not list these. It opens up a possibility for the docs to drift from the JSLib page, and as these increase, we'll have an increasingly difficult time of keeping them from diverging.

File: src/data/markdown/docs/01 guides/02 Using k6/07 Modules.md [hunk 6]
```diff
@@ -384,3 +354,19 @@ Note that on Windows, you also need to make sure that your drive in question, sa
...
+
+These libraries can either be downloaded and included with the test project or loaded directly using HTTP imports as shown [here](#remote-http-s-modules).
+
+## See also
```
Comment (simskij) [inline src/data/markdown/docs/01 guides/02 Using k6/07 Modules.md:369]: Would you mind making an MDX component out of this? Shouldn't be more than maybe 20 lines of code.

We're gonna want to use this in multiple places (we already have, but with difference headings and different ways of listing it) and would benefit from having the format constrained by a component.

File: src/data/markdown/docs/01 guides/02 Using k6/07 Modules.md [hunk 7]
```diff
@@ -26,7 +30,19 @@ with k6, the module itself may only use relative or absolute filesystem imports
...
+}
+```
+
+```js
+//my-test.js
+import { someHelper } from './helpers.js';
+
+export default function () {
+  someHelper();
+}
```
Comment (simskij) [inline src/data/markdown/docs/01 guides/02 Using k6/07 Modules.md:45]: Great addition! 👍

**Review by simskij — CHANGES_REQUESTED** [review]

File: src/data/markdown/docs/01 guides/02 Using k6/07 Modules.md (hunk 6 above)
Comment (ppcano) [reply src/data/markdown/docs/01 guides/02 Using k6/07 Modules.md:369]: 👍🏽

**Pull request merged**